### PR TITLE
feature/1829 - Add Hiroshima Airport (RJOA)

### LINKED ISSUE #1829: Add Hiroshima Airport (RJOA)
This is to add RJOA to the simulation.

- [x]  Airspace
- [x]  Airways
- [x]  Base Airport Properties/Runways
- [x]  SIDs
- [x]  STARs
- [x]  SpawnPatterns
- [x]  Terrain
- [x]  Video Map

### PR DESCRIPTION
**Pull request by JPNite3**
Resolves #1829

### CODE REVIEW

File: documentation/airport-guides/rjoa.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+|"HIROSHIMA RADAR"    | 119.900MHz |                                               |
+|                     | 124.050MHz |                                               |
+|                     | 125.150MHz |                                               |
+|"HIROSHIMA ATIS"     | 127.250MHz |                                               |
+
+## Airspace
+
+There are four radar positions in the Hiroshima Approach Control Area (ACA) — Depature, Approach, and Radar.
+
+As the name suggests, the Departure and Approach positions are for departing and arriving aircrafts to Sendai Airport.
```
Comment (Fechulo) [inline documentation/airport-guides/rjoa.md:29]: I assume that should say Hiroshima, not Sendai

File: documentation/airport-guides/rjoa.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+Departing aircraft will be assigned the MOMOT THREE Depature, or KIJYY TWO Departure in the simulation; they are generally assigned to aircraft departing north/eastbound and west/southbound respectively. Pilots are given an initial climb of FL160. 
+
+###Arrivals
+After handed off from Control, Hiroshima Approach is responsible for: 
+* issuing further descent clearance;
+* vectoring the aircraft to the final approach course if necessary;
+* issuing approach clearance
+* ensuring sufficient spacing between aircraft;
+
+MISEN is the IF (Intermediate Fix), and MACKY the FAF (Final Approach Fix) for ILS 10 approach which should be used in the scenario. Approach should issue a descent clearance so that the aircraft is at 3600ft before reaching YUCCA.
```
Comment (Fechulo) [inline documentation/airport-guides/rjoa.md:45]: I assume that should say MACKY, not YUCCA. It'd also be good to have MACKY in the fixes section.

File: assets/airports/rjoa.json [hunk 3]
```diff
@@ -0,0 +1,758 @@
...
+				["N34d18m04.00", "E132d27m51.00"],
+				["N34d29m56.00", "E132d53m59.00"],
+				["N34d32m06.00", "E133d20m20.00"],
+				["N34d21m47.00", "E133d27m38.00"],
+				["N34d10m50.00", "E133d23m13.00"],
+				["N34d04m27.00", "E132d35m03.00"]
+			]
+        },
+		{
+            "floor": 0,
```
Comment (Fechulo) [inline assets/airports/rjoa.json:72]: This and the following section should be merged into one from the GND to FL80

File: assets/airports/rjoa.json [hunk 4]
```diff
@@ -0,0 +1,758 @@
+{
```
Comment (Fechulo) [inline assets/airports/rjoa.json:1]: Check the indentation throughout this file, it's inconsistent. All indents should be 4 spaces

**Review by Fechulo — CHANGES_REQUESTED** [review]
A few small changes, I'm happy to do them for you, just let me know. Great work!

**Comment by JPNite3** [comment]
> A few small changes, I'm happy to do them for you, just let me know. Great work!

Thanks for the great Christmas gift 😃 I fixed the issues you mentioned so if you could take a look that'd be great!

**Review by Fechulo — APPROVED** [review]
Happy Christmas! 🎅

**Pull request merged**